### PR TITLE
feat(SOP) : Improvements requested by OPS

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
- https://issues.jboss.org/browse/INTLY-2727
- Chat with @Nanyte25 and e.g provided 

## Verification Steps
- See the [Oper SOP](https://github.com/aerogear/mobile-security-service-operator/blob/15301bff54f762f2f926c72d798472cdf2048796/SOP/SOP-operator.adoc)
- See the [MSS SOP](https://github.com/aerogear/mobile-security-service-operator/blob/15301bff54f762f2f926c72d798472cdf2048796/SOP/SOP-mss.adoc)

## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

**Comment by camilamacedo86** [comment]
HI @grdryn and @wei-lee,

Please, could you give a hand here in this review for we are able to move forward with it too? Note that the main content was already checked before. We are more applying here the requests made by SRE.

File: SOP/SOP-mss.adoc [hunk 1]
```diff
@@ -16,92 +16,98 @@ endif::[]
...
-=== MobileSecurityServiceApiHighConcurrentRequests
+. Please following the <<To capture the logs>> procedure before in order to capture the logs and send it to its maintainers.
 
-. Please following the <<General procedure>> procedure before in order to capture the logs and send it to its maintainers.
+TIP: May delete the MobileSecurityService Deployment could allow the operator re-create it with the correct configuration.
 
-== General procedure
+==== MobileSecurityServiceApiHighConcurrentRequests
+
+It means that MSS API has HTTP requests latency longer that 30 seconds.
```
Comment (wei-lee) [inline SOP/SOP-mss.adoc]: the name of the warning suggests that there are too many concurrent requests coming to the server. I am not too sure if the meaning here is right.

File: SOP/SOP-mss.adoc [hunk 2]
```diff
@@ -110,28 +116,47 @@ The following steps would be performed by the Console (OCP UI)
 .. Go to `Applications -> Deployment` and click on in the `mobile-security-service`
 .. Go to the `Environment` tab
 .. Remove the `Env Var` `LOG_LEVEL` and add a new `Env Var` `LOG_LEVEL` with the value `debug`
-.. After saving the re-deploy of the service will be done automatically then you will be able to get further information
+.. After saving the re-deplo will be done automatically.
```
Comment (wei-lee) [inline SOP/SOP-mss.adoc]: re-deploy

File: SOP/SOP-mss.adoc [hunk 3]
```diff
@@ -110,28 +116,47 @@ The following steps would be performed by the Console (OCP UI)
...
+
+. Capture the Service logs
 +
-NOTE: Improve the level of the logs may bring an useful information for its maintainers are able to do improvements and/or fixed required in order to avoid this scenario.
+NOTE: You are able to the the logs by the Console (OCP UI).
```
Comment (wei-lee) [inline SOP/SOP-mss.adoc]: capture the logs

File: SOP/SOP-mss.adoc (hunk 2 above)
Comment (camilamacedo86) [reply SOP/SOP-mss.adoc]: 👍

File: SOP/SOP-mss.adoc (hunk 3 above)
Comment (camilamacedo86) [reply SOP/SOP-mss.adoc]: 👍

File: SOP/SOP-mss.adoc (hunk 1 above)
Comment (camilamacedo86) [reply SOP/SOP-mss.adoc]: It was the name defined by @austincunningham and @davidffrench.
It shows appropriated with the desc here in the impl of this metric: https://github.com/aerogear/mobile-security-service/blob/37006af484e22d2fdecf8429d25f62cddd0b9cee/pkg/web/middleware/middleware.go#L32

**Comment by camilamacedo86** [comment]
Hi @wei-lee the typos changes are applied. However, this repo loses approval. Please, could you do it again for we are able to merge?

**Pull request merged**